classes/site/Site.php

```diff
@@ -235,6 +235,22 @@ public function setSupportedLocales($supportedLocales)
     {
         $this->setData('supportedLocales', $supportedLocales);
     }
+
+    /**
+    * Get the unique site ID.
+    */
+    public function getUniqueSiteID(): string
+    {
+        return $this->getData('uniqueSiteId');
```

Comment:
Even though the migration should handle things properly, I'd keep the pattern from the previous code for safety reasons 😁
I mean the part that checks whether the `uniqueSiteId` value is falsy, then regenerate it on the fly.

Reply:
OK...
I then changed this function so that it returns `?string` and consider that check in the PKPApplication::getUUID()